Proiect_PAO/src/aplicatieBancara/Banca.java

```diff
@@ -0,0 +1,56 @@
+package aplicatieBancara;
+
+public class Banca {
+
+    private static String denumire;
+    private static String sediulCentral;
+    private static String contact;
+    private static String codSwift;
+    private static String prefixIBAN;
+
+
+    private static Banca banca;
+
+    private Banca(){
+        denumire = "Banca Transilvania";
+        sediulCentral = "Cluj-Napoca, Cluj, str. Calea Dorobantilor nr. 30-36, 400117";
+        contact = "0264 308 028";
+        codSwift = "BTRLRO22";
+        prefixIBAN = "RO05BTRLRONCRT";
+    }
```

Comment:
intentia este ca aplicatia sa fie valabila in scenariul acesta doar pentru Banca Transilvania?

Reply:
Da. Dar se poate modifica usor pentru a functiona cu mai multe banci.